documentation/modules/ROOT/pages/modules/logging/p_configuring-log-level-docker.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring-log-level-docker"]
+= Configuring the log level in the {prodname} Docker images
+
+The {prodname} container images for Zookeeper, Kafka, and Kafka Connect all set up their `log4j.properties` file to configure the Debezium-related loggers.
```

Comment:
For downstream we don't provide any container images atm. Are you going to exclude this one downstream?

Reply:
Yes, it will be excluded from the downstream.

Reply:
Ok, cool.